UOP1_Project/Assets/Scripts/PlayerSpawnPoint.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+
+/// <summary>
+/// Spawns the player at its own location
+/// </summary>
+public class PlayerSpawnPoint : MonoBehaviour
```

Comment:
This could be a generic spawner.

Reply:
Correct, but it's named Player only because it's doing some Player-Only relevant stuff.